openrtb_ext/imp_kargo.go

```diff
@@ -0,0 +1,5 @@
+package openrtb_ext
+
+type ImpExtKargo struct {
+	TagId string `json:"tagid"`
```

Comment:
```suggestion
	AdSlot string `json:"adSlot"`
```

The `TagId` is already a standard field on Imp structs. Instead can we make this an AdSlot field that's optional?

Reply:
I'm a bit unclear on this. Do we just want the `adSlotID` instead or the entire `adSlot` as a string?

Reply:
I'll go ahead with `adSlotId` for now